js_modules/dagit/packages/core/src/workspace/WorkspaceContext.tsx

```diff
@@ -31,67 +32,85 @@ type WorkspaceState = {
   loading: boolean;
   locationEntries: RepositoryLocationNode[];
   allRepos: DagsterRepoOption[];
-  refetch: () => Promise<ApolloQueryResult<RootRepositoriesQuery>>;
+  refetch: () => Promise<ApolloQueryResult<unknown>>;
 };
 
 export const WorkspaceContext = React.createContext<WorkspaceState>(
   new Error('WorkspaceContext should never be uninitialized') as any,
 );
 
-const ROOT_REPOSITORIES_QUERY = gql`
-  query RootRepositoriesQuery {
-    workspaceOrError {
+export const ROOT_WORKSPACE_FRAGMENT = gql`
+  fragment RootWorkspaceFragment on Workspace {
+    locationEntries {
       __typename
-      ... on Workspace {
-        locationEntries {
-          __typename
+      id
+      name
+      loadStatus
+      displayMetadata {
+        key
+        value
+      }
+      updatedTimestamp
+      locationOrLoadError {
+        ... on RepositoryLocation {
           id
+          isReloadSupported
+          serverId
           name
-          loadStatus
-          displayMetadata {
-            key
-            value
-          }
-          updatedTimestamp
-          locationOrLoadError {
-            ... on RepositoryLocation {
+          repositories {
+            id
+            name
+            pipelines {
               id
-              isReloadSupported
-              serverId
               name
-              repositories {
+              isJob
+              graphName
+              pipelineSnapshotId
+              modes {
                 id
                 name
-                pipelines {
-                  id
-                  name
-                  isJob
-                  graphName
-                  pipelineSnapshotId
-                  modes {
-                    id
-                    name
-                  }
-                }
-                partitionSets {
-                  id
-                  mode
-                  pipelineName
-                }
-                ...RepositoryInfoFragment
+              }
+              schedules {
```

Comment:
`schedules` and `sensors` were added to this from the LeftNav query, otherwise they were the same.